MvvmCross/Core/Core/Navigation/MvxNavigationService.cs

```diff
@@ -119,21 +120,23 @@ private static bool TryGetRoute(string url, out KeyValuePair<Regex, Type> entry)
             OnAfterNavigate(this, args);
         }
 
-        private async Task NavigateRoute(string path)
+        private async Task<IMvxViewModel> NavigateRoute(string path)
         {
             KeyValuePair<Regex, Type> entry;
```

Comment:
I have no idea. This is from another earlier PR that included the url navigation.